pylxd/image.py

```diff
@@ -80,7 +81,66 @@ def create(cls, client, image_data, public=False, wait=False):
             Operation.wait_for_operation(client, response.json()['operation'])
         return cls(client, fingerprint=fingerprint)
 
+    @classmethod
+    def create_from_config(cls, client, config, wait=False):
```

Comment:
This method is for the `copy_to_lxd` method, `copy_to_lxd` generates the `config` for this and calls 
`newclient.images.create_from_config(`generated_config`, wait=`given wait`)

It would also allow someone to create images from url and simplestreams, but for that we want implicit methods i think.